### DISCUSSION TITLE
are you able to get error message when running query in the dash besides "invalid sql query"?

### DISCUSSION THREAD

**Discussion by devhandler** [supabase/supabase#15131] [Questions]
not sure how to get more detailed error info, but a plain "invalid sql" won't help much especially when the query is long

**Comment by GaryAustin1** [comment]
See here for improvement request to the editor:   https://github.com/orgs/supabase/discussions/14206

Mean while look into 3rd party tools like Datagrip, or even online syntax checkers... or ask for help in the Supabase Discord.
